src/backends/tensorflow/autofill.cc

```diff
@@ -295,7 +295,8 @@ AutoFillSavedModel::Create(
       TRTISTF_NO_GPU_DEVICE, false /* have_graph */, 0 /* graph_level */,
       graphdef_backend_config->allow_gpu_memory_growth,
       graphdef_backend_config->per_process_gpu_memory_fraction,
-      graphdef_backend_config->allow_soft_placement));
+      graphdef_backend_config->allow_soft_placement,
+      graphdef_backend_config->memory_limit_mb));
```

Comment:
This can't be just a single value. The user is going to need to be able to specify a value for each physical GPU. We can update this as part of the other ticket.

Reply:
memory_limit_mb should be a vector of vectors of floats. Could we parse this from the user somehow?

Reply:
Changed to map from int to vector of floats